pkg/debug/portforward.go

```diff
@@ -0,0 +1,44 @@
+package debug
+
+import (
+	"net/http"
+	"net/url"
+
+	"github.com/openshift/odo/pkg/occlient"
+	"k8s.io/client-go/tools/portforward"
+	"k8s.io/client-go/transport/spdy"
+	k8sgenclioptions "k8s.io/kubernetes/pkg/kubectl/genericclioptions"
+)
+
+type PortForwarder interface {
```

Comment:
There should be a comment explaining what is a purpose of this interface

Reply:
added

Reply:
That comment doesn't provide much more clarification than the title itself :-(

Reply:
will remove the interface

Reply:
done removed